src/App.css

```
@@ -1355,6 +1355,52 @@ body,
+
+.transcript-segment-timestamp {
+	font-family: ui-monospace, SFMono-Regular, Menlo, monospace;
+	font-size: 0.8rem;
+	color: #666;
+}
+
+.transcript-segment-text {
+	font-size: 0.95rem;
+	line-height: 1.6;
+	color: #333;
+	word-wrap: break-word;
```

Stylelint (17.10.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)